en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-250-to-300.md

```diff
@@ -3,350 +3,1678 @@
 The following information describes how to upgrade your API Manager server **from APIM 2.5.0 to 3.0.0** .
 
 !!! note
-To upgrade **from a version older than 1.8.0** , follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
+    To upgrade **from a version older than 1.8.0** , follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
 
-!!! tip
-    Before you begin
+!!! attention "Before you Begin"
 
-    - This release is a WUM-only release. This means that there are no manual patches. Any further fixes or latest updates for this release can be updated through the WSO2 Update Manager (WUM).
+    1. This release is a WUM-only release. This means that there are no manual patches. Any further fixes or latest updates for this release can be updated through the WSO2 Update Manager (WUM).
 
         -   **If you are upgrading to this version, in order to use this version in your production environment** , use the WSO2 Update Manager and get the latest available updates for WSO2 API Manager 3.0.0. For more information on how to do this, see [Updating WSO2 Products](https://docs.wso2.com/display/ADMIN44x/Getting+Started+with+WUM).
```

Comment:
Shall we use the updated URL instead of https://docs.wso2.com/display/ADMIN44x/Getting+Started+with+WUM?

Reply:
Changed the URL as https://docs.wso2.com/display/updates/Using+WSO2+Update+Manager